### DISCUSSION TITLE
Quasar using dangerous NPM package

### DISCUSSION THREAD

**Discussion by ghost** [quasarframework/quasar#17283] [General - Components / Directives / etc] [answered]
@quasar/app-vite@1.9.3 uses glob@7.2.3 which in turn uses inflight@1.0.6 which is flagged as deprecated.  The reason for the deprecation warning is that the package is no longer supported, and it is known to leak memory.

Is it possible to use Quasar with Vite, without the risk posed by using deprecated packages that leak memory?  If so, how?

**Comment by stefanvanherwijnen** [comment, marked as answer]
This should be fixed in v2 (`archiver` dependency is updated)(currently in beta):
https://github.com/quasarframework/quasar/blob/dev/app-vite/package.json
https://github.com/quasarframework/quasar/releases/tag/%40quasar%2Fapp-vite-v2.0.0-beta.13

**Reply by ghost** [reply]
Thanks for the quick response and clarification.
